cocos/physics-2d/framework/physics-internal-types.ts

```diff
@@ -1,4 +1,4 @@
 export declare class DelayEvent {
-    target: Object;
-    func: Function;
+    target: unknown;
```

Comment:
Can you explain the reason to modify this to unknown

Reply:
2 reasons:

1, because `DelayEvent` is used for pending execution for `PhysicsSystem` only, the `target` will be the function scope to be applied.
for example:

    PhysicsSystem2D.instance._callAfter(func, scope);
    // and _callAfter:
    _callAfterStep (target: unknown, func: () => void) {
        if (this._steping) {
            this._delayEvents.push({
                target,
                func,
            });
        } else {
            func.call(target);
        }
    }

2, the main reason: ESLint does not allow `Object`, `Function` to be a type. so the ESLint test on this PR can't be passed....
    according to the scope parameter type the `Function.prototype.apply / call` need, they are defined as this:

    apply<T, A extends any[], R>(this: (this: T, ...args: A) => R, thisArg: T, args: A): R;
    call<T, A extends any[], R>(this: (this: T, ...args: A) => R, thisArg: T, ...args: A): R;

so, if not use `unknown`, better to modify this `DelayEvent` as `DelayEvent<TScope>` etc solution, in my opinion this `DelayEvent` should not be a class. an `interface` is enough.